File: latihan 1/latihan pkbn/pkbn.py
nama = input("Masukan nama barang: ")
harga = float(input("Masukan harga barang: "))
jumlah = int(input("Masukan jumlah barang: "))

total = harga * jumlah

if total > 13200:
    diskon = total * 0.02
else:
    diskon = 0.2

total_bayar = total - diskon 

print("\n===== STRUK PEMBELIAN =====")
print(f"Nama barang   : {nama}")
print(f"Harga satuan  : {harga}")
print(f"Jumlah beli   : {jumlah}")
print(f"Total harga   : {total}")
print(f"Diskon        : {diskon}")
print(f"Total bayar   : {total_bayar}")
print("=============================")

File: latihan 1/latihan pkbn/pkbn.cpp
#include <iostream>
#include <iomanip>
#include <string>
using namespace std;

int main() {
    string nama;
    double harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    getline(cin, nama);

    cout << "Masukkan harga barang: ";
    cin >> harga;

    cout << "Masukkan jumlah beli: ";
    cin >> jumlah;

    double total = harga * jumlah;
    double diskon = 0;

    // Logika diskon
    if (total > 13200) {
        diskon = total * 0.02; // Diskon 4%
    } else {
        diskon = 02; // Tidak ada diskon
    }

    double totalBayar = total - diskon;

    // Output struk
    cout << "\n==== STRUK PEMBELIAN ====\n";
    cout << "Nama Barang   : " << nama << endl;
    cout << fixed << setprecision(2);
    cout << "Harga Satuan  : " << harga << endl;
    cout << "Jumlah Beli   : " << jumlah << endl;
    cout << "Total Harga   : " << total << endl;
    cout << "Diskon        : " << diskon << endl;
    cout << "Total Bayar   : " << totalBayar << endl;
    cout << "==========================\n";

    return 0;
}